### PR TITLE
feat: Add new daedalus.xml mavlink messages

### PR DESCRIPTION
**Pull request by cb0s**
- [x] I have signed the individual contributor's license agreement **and sent** it to the board of the WüSpace e. V. organization.

---
This change is [Reviewable](https://reviewable.io/reviews/wuespace/telestion-project-daedalus2/75)

### CODE REVIEW

File: definitions/daedalus.xml [hunk 1]
```diff
@@ -1,68 +1,74 @@
...
+		<field type="float" name="filter_vel_vertical_ind">induced vertical velocity (lift)  in m/s</field>
+		<field type="float" name="controller_blade_pitch">setpoint for the pitch angle for the servos </field>
+		<field type="float" name="controller_fin_angle">desired fin angle </field>
+		<field type="uint8_t" name="controller_id">indentifier for controller type/state</field>
+		<field type="uint8_t" name="available_status">imu_acc_avail, imu_gyro_avail, baro_avail, vacuum_baro_avail, tacho_rot_avail, servo_amps_avail, bat_temp_avail, volts_avail in this order with individual size of 1 bit</field>
+		</message>
+		
+		<message id="10004" name="CON_CMD">
+		<description> Freetext command input, one command per message.
+		</description>
```
Comment (fussel178) [inline definitions/daedalus.xml:70]: Little style change :wink: 

```suggestion
		<description>Freetext command input, one command per message.</description>

```

**Review by fussel178 — CHANGES_REQUESTED** [review]
One little change, otherwise LGTM on the "style" and "repo" side.
For the implementation and coordination side please refer to @pklaschka :see_no_evil:

**Review by pklaschka — APPROVED** [review]
> Reviewed 1 of 1 files at r1.
> *[Reviewable](https://reviewable.io/reviews/wuespace/telestion-project-daedalus2/75)* status: all files reviewed, 1 unresolved discussion (waiting on @cb0s and @jvpichowski)

**Review by pklaschka — APPROVED** [review]
:lgtm:

> Reviewed 1 of 1 files at r2.
> *[Reviewable](https://reviewable.io/reviews/wuespace/telestion-project-daedalus2/75#-:-McsHI_p-yRdXxAN3pMx:bnfp4nl)* status: all files reviewed, 1 unresolved discussion (waiting on @cb0s and @jvpichowski)

**Review by fussel178 — APPROVED** [review]
LGTM. Thank you for your contribution :palm_tree:

**Pull request merged**